scripts/karma.coffee

```diff
@@ -23,13 +23,14 @@ scores = [
 getUserKarmaExpression = (userKarmaToken) ->
   hasAtSymbol = userKarmaToken.startsWith('@')
   username = if hasAtSymbol
-    then userKarmaToken.slice(1, -2)
-    else userKarmaToken.slice(0, -2)
+  then userKarmaToken.slice(1, -2)
+  else userKarmaToken.slice(0, -2)
 
   operator = userKarmaToken.slice(-2)
 
   {
-    hasAtSymbol,
+    type: 'UserKarmaExpression',
```

Comment:
Is this used for anything? I don't think we need it.

Reply:
No, not being used anywhere in the code.

It's just a bit of metadata that could be useful if there were more token expressions in the future.